docker-storage-setup.sh

```diff
@@ -1190,11 +1196,6 @@ done
 if [ $# -eq 2 ]; then
     STORAGE_IN_FILE=$1
     STORAGE_OUT_FILE=$2
-else
-    if [ $# -gt 0 ]; then
-	usage $(basename $0) >&2
-	exit 1
-    fi
 fi
```

Comment:
Why are you taking this check away. We want to make sure that either user need to specify both input and output file or does not specify any.